### PR TITLE
min: Add version 1.24.0

### LINKED ISSUE #8354: [Request] Min
### Information

**Name:** Min

**Description:** A fast, minimal browser that protects your privacy

**Homepage:** https://github.com/minbrowser/min

**Download link(s):** https://github.com/minbrowser/min/releases/latest/download/Min-v1.24.0-win32-x64.zip

**Some indication of popularity/repute:** 4.9k GitHub stars

### PR DESCRIPTION
**Pull request by nohopestage**
Closes #8354

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
If an app does not natively store data in its own directory, we shouldn't try to force persistence. (The chrome manifests are an exception to this because of backward compatibility). 

Storing data in AppData is fine.

**Comment by nohopestage** [comment]
This is exactly how it's done in [ungoogled-chromium](https://github.com/lukesampson/scoop-extras/tree/master/bucket/ungoogled-chromium.json), and Min is basically chromium.

**Comment by rashil2000** [comment]
> (The older chrome manifests are an exception to this because of backward compatibility)

**Comment by nohopestage** [comment]
Removed persistence, the app now stores its data in AppData.

File: bucket/min.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "1.24.0",
+    "description": "A fast, minimal browser that protects your privacy",
+    "homepage": "https://github.com/minbrowser/min",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/minbrowser/min/releases/download/v1.24.0/Min-v1.24.0-win32-x64.zip",
+            "hash": "e00a9d6e7652fd4dd40dda83fb8b316016f21503f25f2a1cb9be88d21c9e5c65"
```
Comment (rashil2000) [inline bucket/min.json:9]:
```suggestion
            "hash": "e00a9d6e7652fd4dd40dda83fb8b316016f21503f25f2a1cb9be88d21c9e5c65",
            "extract_dir": "Min-v1.24.0"
```

File: bucket/min.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "Move-Item \"$dir\\Min*\\*\" \"$dir\\bin\"",
+            "Remove-Item \"$dir\\Min*\""
+        ]
+    },
+    "bin": [
+        [
+            "bin\\Min.exe",
+            "min"
+        ]
+    ],
```
Comment (rashil2000) [inline bucket/min.json:26]:
```suggestion
    "bin": "Min.exe",
```

File: bucket/min.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "shortcuts": [
+        [
+            "bin\\Min.exe",
+            "Min"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/minbrowser/min/releases/latest/download/Min-v$version-win32-x64.zip"
```
Comment (rashil2000) [inline bucket/min.json:35]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/minbrowser/min/releases/latest/download/Min-v$version-win32-x64.zip"
                "extract_dir": "Min-v$version"
            }
        }
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Pull request merged**